assets/src/edit-story/components/canvas/multiSelectionMovable.js

```diff
@@ -245,14 +251,15 @@ function MultiSelectionMovable({ selectedElements }) {
         onGroupEventEnd({ targets, isResize: true });
       }}
       renderDirections={CORNER_HANDLES}
-      snappable={true}
-      snapElement={true}
-      snapHorizontal={true}
-      snapVertical={true}
-      snapCenter={true}
-      horizontalGuidelines={[0, canvasHeight / 2, canvasHeight]}
-      verticalGuidelines={[0, canvasWidth / 2, canvasWidth]}
+      snappable={canSnap}
+      snapElement={canSnap}
+      snapHorizontal={canSnap}
+      snapVertical={canSnap}
+      snapCenter={canSnap}
+      horizontalGuidelines={canSnap ? [0, canvasHeight / 2, canvasHeight] : []}
```

Comment:
I believe the `canSnap ?` shouldn't be needed here.

Reply:
So we want to keep the horizontal guides even when command is pressed?

Reply:
I'm ok if it works. Last time I tried, the guidelines couldn't be changed once dragging has started.

Reply:
It works apparently!